### ISSUE TITLE
Segfault when using tiescalar on a scalar and use it as a glob

### ISSUE THREAD

**Issue by atoomic** [rurban/perl-compiler#279]
tested with 5.22
working with 5.14

````
> perl -e '*TIESCALAR = sub {}; tie my $var => "main", 42; <${var}>; print qq/ok\n/'
ok
> perlcc -e '*TIESCALAR = sub {}; tie my $var => "main", 42; <${var}>; print qq/ok\n/'; ./a.out
pccWYpCJ.c:8220: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8220: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8221: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8221: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8222: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8222: warning: this decimal constant is unsigned only in ISO C90
pccWYpCJ.c:8670: warning: integer constant is too large for ‘unsigned long’ type
pccWYpCJ.c:8671: warning: integer constant is too large for ‘unsigned long’ type
Segmentation fault
````

````
Program received signal SIGSEGV, Segmentation fault.
0x002dc354 in Perl_pp_gv () at pp_hot.c:102
102	    if (isGV(cGVOP_gv)
````

**Comment by rurban** [comment]
Excellent testcase. We don't initialize the GV anymore.

**Comment by toddr** [comment]
op/tie_fetch_count.t is somehow fixed on cm522. bisecting.

**Comment by toddr** [comment]
it appears to have been fixed by ea4adc0690c8afdf975b0164e5b773f1d6157475

**Comment by toddr** [comment]
This is resolved by #233
